blog/static/js/main.js

```diff
@@ -0,0 +1,110 @@
+document.getElementById("titlelink")
+
+let element = document.getElementById("titlelink");
+
+element.style["color"] = "aqua";
+element.style["border"] = "3px solid aqua";
+
+function doTheStuff() { //defining a python function
```

Comment:
Could you
 - rename this function to something more descriptive?
 - add a comment explaining what this section of code does?